iocore/net/quic/QUICStream.cc

```diff
@@ -408,7 +408,14 @@ QUICStream::generate_frame(QUICEncryptionLevel level, uint64_t connection_credit
 {
   SCOPED_MUTEX_LOCK(lock, this->_write_vio.mutex, this_ethread());
 
-  QUICFrameUPtr frame = QUICFrameFactory::create_null_frame();
+  // retransmitted frame has higher priority.
+  QUICFrameUPtr frame = this->pop_from_retransmitted_queue();
```

Comment:
I don't think we can do this because `maximum_frame_size` can be smaller than the frame popped. We don't need to think about a window for flow control in this case, but we still need to think about a size of packet. MTU may be changed. You wrote `split()` for it, right?